chunky/src/java/se/llbit/chunky/renderer/projection/OmniDirectionalStereoProjector.java

```diff
@@ -23,7 +23,9 @@
 
 /**
  * A projector for Omni-Directional Stereo (ODS) images. x is mapped to yaw, y is mapped to
- * pitch.
+ * pitch. This projector is like the {@link PanoramicProjector} but can create distinct
+ * images for the left and the right eye to create panoramic stereo images that are perfect
+ * for viewing on VR devices.
```

Comment:
Looks good except I'd change "displacing the view rays" to "displacing the view ray origins"